lib/api/controllers/documentController.js

```diff
@@ -432,7 +433,12 @@ class DocumentController extends BaseController {
         }
 
         if (errors.length > 0) {
-          request.setError(errorsManager.get('deletion_failed', errors));
+          request.setError(
+            errorsManager.get(
+              'services',
+              'storage',
+              'incomplete_delete',
+              errors));
         }
 
         // @todo should be done in clientAdapter ?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)